lib/galaxy/tools/__init__.py

```diff
@@ -304,7 +304,7 @@ class safe_update(NamedTuple):
     "Filter1": safe_update(parse_version("1.1.0"), parse_version("1.1.1")),
     "__BUILD_LIST__": safe_update(parse_version("1.0.0"), parse_version("1.1.0")),
     "__APPLY_RULES__": safe_update(parse_version("1.0.0"), parse_version("1.1.0")),
-    "__EXTRACT_DATASET__": safe_update(parse_version("1.0.0"), parse_version("1.0.1")),
+    "__EXTRACT_DATASET__": safe_update(parse_version("1.0.0"), parse_version("1.0.2")),
```

Comment:
hrm. maybe this is not a safe update :)

Reply:
yes, that isn't safe and I don't think an update would ever be, so I think I don't want to do this at all